.travis.yml

```diff
@@ -8,6 +13,13 @@ addons:
     packages:
       # For building MUSL static builds on Linux.
       - musl-tools
+      - cmake
```

Comment:
Are these new packages all for `cargo-travis`?

Reply:
Yes they are.